Modules/Common/src/BigScreen.cxx

```diff
@@ -33,69 +31,79 @@ namespace o2::quality_control_modules::common
 void BigScreen::configure(const boost::property_tree::ptree& config)
 {
   mConfig = BigScreenConfig(getID(), config);
+}
 
-  mColors[Quality::Null.getName()] = kViolet - 6;
-  mColors[Quality::Bad.getName()] = kRed;
-  mColors[Quality::Medium.getName()] = kOrange - 3;
-  mColors[Quality::Good.getName()] = kGreen + 2;
+//_________________________________________________________________________________________
 
-  float dx = 1.0 / mConfig.mNCols;
-  float dy = 1.0 / mConfig.mNRows;
-  float paveHalfWidth = 0.3 * dx;
-  float paveHalfHeight = 0.3 * dy;
+std::string getParameter(o2::quality_control::core::CustomParameters customParameters,
+                         std::string parName,
+                         const o2::quality_control::core::Activity& activity)
+{
+  std::string parValue;
+  auto parOpt = customParameters.atOptional(parName, activity);
+  if (parOpt.has_value()) {
+    parValue = parOpt.value();
+  } else {
+    parOpt = customParameters.atOptional(parName);
+    if (parOpt.has_value()) {
+      parValue = parOpt.value();
+    }
+  }
+  return parValue;
```

Comment:
@Barthelemy it seems like a lot of code to use the CustomParameters. Could you have a look if it could not be done with less?

Reply:
This is how I wrapped the CustomParameters calls in the MUON code: https://github.com/AliceO2Group/QualityControl/blob/ddb85738c291bac84c38eb7e396a974ea3b74ec4/Modules/MUON/Common/include/MUONCommon/Helpers.h#L36